crypto/kernel/err.c

```diff
@@ -67,12 +67,32 @@ srtp_err_status_t srtp_err_reporting_init ()
     return srtp_err_status_ok;
 }
 
+static srtp_err_report_handler_func_t * srtp_err_report_handler = NULL;
+
+srtp_err_status_t srtp_install_err_report_handler(srtp_err_report_handler_func_t func)
+{
+    srtp_err_report_handler = func;
+}
```

Comment:
Missing some sort of return here. Probably just an `srtp_err_status_ok` since there is little that can go wrong? Or make it void. Not sure. :-)